src/python/turicreate/toolkits/object_detector/_tf_image_augmenter.py

```diff
@@ -8,6 +8,8 @@
 from __future__ import absolute_import as _
 
 import numpy as np
+from PIL import Image
+import PIL
```

Comment:
Why do we need to import `PIL`? Would the line above this one suffice?

Reply:
good!